### ISSUE TITLE
Fastlane supply init fails to download release notes from production track

### ISSUE THREAD

**Issue by TheDancerCodes** [fastlane/fastlane#16548]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
When I run the `fastlane supply init` command, it is expected that all the current Google Play Store metadata will be downloaded to `fastlane/metadata/android`.

This is not the case as the command is having challenges retrieving the release notes but successfully retrieves the rest of the metadata.

It fails with this error: `[!] Version '1.0.1' for 'com.thedancercodes.fastlanedroid.todoapp' does not seem to have any release notes. Nothing to download. (FastlaneCore::Interface::FastlaneError)
` although the application is live, contains release notes and has been rolled out to the public. 🤷‍♂️ 

I have also correctly followed all the steps to set up supply from the [fastlane docs](https://docs.fastlane.tools/getting-started/android/setup/#setting-up-supply)

##### Command executed
`fastlane supply init`

##### Complete output when running fastlane, including the stack trace and command used

```
 $ fastlane supply init --verbose
[✔] 🚀 
Successfully loaded Appfile at path '/Users/xxx/Projects/Android/fastlane-droid/fastlane/Appfile'
- json_key_file: '/Users/xxx/Projects/Android/fastlane-droid/google-play-json-key-file.json'
- package_name: 'com.thedancercodes.fastlanedroid.todoapp'
-------
INFO [2020-06-02 13:02:26.29]: 🕗  Downloading metadata, images, screenshots...
DEBUG [2020-06-02 13:02:26.31]: Fetching a new access token from Google...
INFO [2020-06-02 13:02:28.27]: 📝  Downloading metadata (en-US)
INFO [2020-06-02 13:02:28.27]: Writing to fastlane/metadata/android/en-US/title.txt...
INFO [2020-06-02 13:02:28.27]: Writing to fastlane/metadata/android/en-US/short_description.txt...
INFO [2020-06-02 13:02:28.27]: Writing to fastlane/metadata/android/en-US/full_description.txt...
INFO [2020-06-02 13:02:28.27]: Writing to fastlane/metadata/android/en-US/video.txt...
INFO [2020-06-02 13:02:28.27]: 🖼️  Downloading images (en-US)
INFO [2020-06-02 13:02:28.27]: Downloading `featureGraphic` for en-US...
DEBUG [2020-06-02 13:02:29.15]: Initial URL received: 'https://lh3.googleusercontent.com/qEII3X8zRL5m7kC-I6yoTz4bWG3GWnvcIOhte1y5FW7buygyXQhNBEKZlmGgzlxg1hU'
DEBUG [2020-06-02 13:02:29.15]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:29.15]: URL after removing params: 'httpslh3.googleusercontent.com443/qEII3X8zRL5m7kC-I6yoTz4bWG3GWnvcIOhte1y5FW7buygyXQhNBEKZlmGgzlxg1hU'
INFO [2020-06-02 13:02:30.07]:  Downloaded - fastlane/metadata/android/en-US/images/featureGraphic.png
INFO [2020-06-02 13:02:30.07]: Downloading `icon` for en-US...
DEBUG [2020-06-02 13:02:30.50]: Initial URL received: 'https://lh3.googleusercontent.com/38zBGzDcWZHH8-bEiZWhfzekPEdJHi2PsL2UDBqoJGNNCm5dwzllQaS6FCQCH9Ng1Tw'
DEBUG [2020-06-02 13:02:30.50]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:30.50]: URL after removing params: 'httpslh3.googleusercontent.com443/38zBGzDcWZHH8-bEiZWhfzekPEdJHi2PsL2UDBqoJGNNCm5dwzllQaS6FCQCH9Ng1Tw'
INFO [2020-06-02 13:02:31.40]:  Downloaded - fastlane/metadata/android/en-US/images/icon.png
INFO [2020-06-02 13:02:31.40]: Downloading `promoGraphic` for en-US...
INFO [2020-06-02 13:02:31.81]: Downloading `tvBanner` for en-US...
INFO [2020-06-02 13:02:32.21]: Downloading `phoneScreenshots` for en-US...
DEBUG [2020-06-02 13:02:32.69]: Initial URL received: 'https://lh3.googleusercontent.com/2qOVTpg2igQqgH4cSsFjD5YuYZfmKBJIgXoBaUdhx3UEsndURLLeQUvGv8qaY4T7sg'
DEBUG [2020-06-02 13:02:32.69]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:32.69]: URL after removing params: 'httpslh3.googleusercontent.com443/2qOVTpg2igQqgH4cSsFjD5YuYZfmKBJIgXoBaUdhx3UEsndURLLeQUvGv8qaY4T7sg'
DEBUG [2020-06-02 13:02:32.69]: Initial URL received: 'https://lh3.googleusercontent.com/jif4FKqJypy8ci6aHfk1047ho2f5GhWj3qEcG_1wqKjjC4NkGM8Uuob1gXb-Cp8rI6w'
DEBUG [2020-06-02 13:02:32.69]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:32.69]: URL after removing params: 'httpslh3.googleusercontent.com443/jif4FKqJypy8ci6aHfk1047ho2f5GhWj3qEcG_1wqKjjC4NkGM8Uuob1gXb-Cp8rI6w'
DEBUG [2020-06-02 13:02:32.69]: Initial URL received: 'https://lh3.googleusercontent.com/f-BIMQ_Ina8cIJkBow3zimo1Bx1a2HLxDS0OXL_fkMwbNyvql3yX_s-1nw2ymO9koF0'
DEBUG [2020-06-02 13:02:32.69]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:32.69]: URL after removing params: 'httpslh3.googleusercontent.com443/f-BIMQ_Ina8cIJkBow3zimo1Bx1a2HLxDS0OXL_fkMwbNyvql3yX_s-1nw2ymO9koF0'
DEBUG [2020-06-02 13:02:32.69]: Initial URL received: 'https://lh3.googleusercontent.com/XdchaDGjwZDA0NiFjC9kps-_-3NidT076I4wJIqD6cdh28EY1ZV6ZxVNygurTfOSTCw'
DEBUG [2020-06-02 13:02:32.69]: Removed params ('') from the URL
DEBUG [2020-06-02 13:02:32.69]: URL after removing params: 'httpslh3.googleusercontent.com443/XdchaDGjwZDA0NiFjC9kps-_-3NidT076I4wJIqD6cdh28EY1ZV6ZxVNygurTfOSTCw'
INFO [2020-06-02 13:02:33.35]:  Downloaded - fastlane/metadata/android/en-US/images/phoneScreenshots/1_en-US.png
INFO [2020-06-02 13:02:34.19]:  Downloaded - fastlane/metadata/android/en-US/images/phoneScreenshots/2_en-US.png
INFO [2020-06-02 13:02:34.83]:  Downloaded - fastlane/metadata/android/en-US/images/phoneScreenshots/3_en-US.png
INFO [2020-06-02 13:02:35.46]:  Downloaded - fastlane/metadata/android/en-US/images/phoneScreenshots/4_en-US.png
INFO [2020-06-02 13:02:35.46]: Downloading `sevenInchScreenshots` for en-US...
INFO [2020-06-02 13:02:35.83]: Downloading `tenInchScreenshots` for en-US...
INFO [2020-06-02 13:02:36.36]: Downloading `tvScreenshots` for en-US...
INFO [2020-06-02 13:02:36.76]: Downloading `wearScreenshots` for en-US...
INFO [2020-06-02 13:02:39.02]: Found '1.0.1' in 'production' track.
Traceback (most recent call last):
        17: from /Users/xxx/.rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `<main>'
        16: from /Users/xxx/.rvm/gems/ruby-2.5.5/bin/ruby_executable_hooks:24:in `eval'
        15: from /Users/xxx/.rvm/gems/ruby-2.5.5/bin/fastlane:23:in `<main>'
        14: from /Users/xxx/.rvm/gems/ruby-2.5.5/bin/fastlane:23:in `load'
        13: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/bin/fastlane:23:in `<top (required)>'
        12: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
        11: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/supply/lib/supply/commands_generator.rb:12:in `start'
        10: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/supply/lib/supply/commands_generator.rb:60:in `run'
         9: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
         8: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
         7: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
         6: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
         5: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
         4: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/supply/lib/supply/commands_generator.rb:54:in `block (2 levels) in run'
         3: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/supply/lib/supply/setup.rb:27:in `perform_download'
         2: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/supply/lib/supply/client.rb:251:in `release_listings'
         1: from /Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
/Users/xxx/.rvm/gems/ruby-2.5.5/gems/fastlane-2.148.1/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!': [!] Version '1.0.1' for 'com.thedancercodes.fastlanedroid.todoapp' does not seem to have any release notes. Nothing to download. (FastlaneCore::Interface::FastlaneError)

```

<details>
Whenever I run fastlane supply init, the step seems to be failing with the error:

`[!] Version '3.0.1' for <myPackage> does not seem to have any release notes. Nothing to download. `
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.4                                     |
| Ruby                        | 2.5.5                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.24.1 (Apple Git-126)          |
| Installation Source         | ~/.rvm/gems/ruby-2.5.5/bin/fastlane         |
| Host                        | Mac OS X 10.15.4 (19E287)                   |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.5.5/lib                |
| OpenSSL Version             | OpenSSL 1.1.1d  10 Sep 2019                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.4                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:android)

platform :android do
  
  # Runs before all tasks
  before_all do

   # Add SLACK_URL & make it accessible to all lanes
   ENV["SLACK_URL"]
  end

  # This lane builds our app with specified buildVariant(buildFlavor + BuildType)
  desc "Assemble Build"
  lane :assemble_build do |options|
    gradle(
      task: "assemble",
      flavor: options[:build_flavor],
      build_type: options[:build_type]
    )
  end

  desc "Run unit tests with specified buildVariant(buildFlavor + BuildType)"
  lane :unit_tests do |options|
    gradle(
      task: "test",
      flavor: options[:build_flavor],
      build_type: options[:build_type]
    )
  end

  desc "Run Instrumented tests in Firebase Test Lab"
  lane :instrumentation_tests_firebase_testlab do |options|
    assemble_build(build_flavor:options[:build_flavor], build_type:"Debug")
    @app_apk = Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]

    assemble_build(build_flavor:options[:build_flavor] + "Debug", build_type:"AndroidTest")
    @android_test_apk = Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]

    run_tests_firebase_testlab(
      project_id: ENV["FIREBASE_PROJECT_ID"],
      app_apk: @app_apk,
      android_test_apk: @android_test_apk,
      devices: [
        {
            model: "Nexus6P",
            version: "27"
        }
      ],
      delete_firebase_files: true
    )
  end

  desc "Build a debug APK"
  lane :debug_build do
    gradle(task: "clean assembleDebug")
  end

 # Note: To run this locally, add flavor & update keystore.jks to ENV['KEYSTORE_PATH']
  desc "Build a signed release APK & deploy to slack"
  lane :slack_apk_build do
    gradle(
      task: "assemble",
      build_type: "Release",
      print_command: false,
      properties: {
        "android.injected.signing.store.file" => "keystore.jks",
        "android.injected.signing.store.password" => ENV['STORE_PASSWORD'],
        "android.injected.signing.key.alias" => ENV['KEY_ALIAS'],
        "android.injected.signing.key.password" => ENV['KEY_PASSWORD'],
      }
    )
    upload_to_slack()
    slack(message: ":rocket: *Build Successful!* :rocket:")
  end

  desc "Upload APK to Slack channel"
  private_lane :upload_to_slack do |options|
    file_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
    file_name = file_path.gsub(/\/.*\//,"")
    access_token = ENV["ACCESS_TOKEN"]
    channel_name = "apks"

    sh "echo Uploading " + file_name + " to Slack"
    sh "curl https://slack.com/api/files.upload -F token=\"" + access_token + "\" -F channels=\"" + channel_name + "\" -F title=\"" + file_name + "\" -F filename=\"" + file_name + "\" -F file=@" + file_path
  end

  # Runs after all tasks. This block is called only when the executed lane is successful
  after_all do |lane|

    # Slack action
    # slack(message: ":rocket: Successful Deployment! :rocket:")
  end

  error do |lane, exception|
    slack(
      message: exception.message,
      success: false
      )
  end

  desc "Deploy latest Beta build to Crashlytics Beta"
  lane :beta do
    gradle(task: "assembleRelease")
    crashlytics
  end

  # use like this `fastlane distribute group:only-me`
  desc "Deploy latest Beta build to Firebase App Distribution"
  lane :distribute do |options|
    if options[:group]
      build_android_app(
        task: "assemble",
        flavor: "prod",
        build_type: "release"
      )

      firebase_app_distribution(
          app: ENV['FIREBASE_APP_ID_ANDROID'],
          groups: options[:group],
          release_notes: "Fastlane Droid Release Notes Updated 1.0.1",
          firebase_cli_path: "/usr/local/bin/firebase",
          firebase_cli_token: ENV['FIREBASE_CI_TOKEN']
      )
    else
      UI.user_error!("Missing group name!")
    end
  end
  
  desc "Deploy latest version to Google Play"
  lane :playstore do |options|
    gradle(
      task: "assemble",
      flavor: options[:build_flavor],
      build_type: options[:build_type]
    )
    upload_to_play_store # Uploads the APK built in the gradle step above and releases it to all production users
  end  
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
json_key_file(ENV["GOOGLE_PLAY_JSON_KEY_FILE_PATH"]) # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
package_name("com.thedancercodes.fastlanedroid.todoapp") # e.g. com.krausefx.app

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.148.1 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                     | Version | Update-Status |
| ------------------------------------------ | ------- | ------------- |
| fastlane-plugin-firebase_app_distribution  | 0.1.4   | ✅ Up-To-Date  |
| fastlane-plugin-run_tests_firebase_testlab | 0.3.1   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                        | Version      |
| ------------------------------------------ | ------------ |
| did_you_mean                               | 1.2.0        |
| executable-hooks                           | 1.6.0        |
| bundler-unload                             | 1.0.2        |
| rubygems-bundler                           | 1.4.5        |
| bundler                                    | 2.0.2        |
| CFPropertyList                             | 3.0.2        |
| public_suffix                              | 2.0.5        |
| addressable                                | 2.7.0        |
| atomos                                     | 0.1.3        |
| aws-eventstream                            | 1.1.0        |
| aws-partitions                             | 1.322.0      |
| aws-sigv4                                  | 1.1.4        |
| jmespath                                   | 1.4.0        |
| aws-sdk-core                               | 3.97.0       |
| aws-sdk-kms                                | 1.32.0       |
| aws-sdk-s3                                 | 1.67.0       |
| babosa                                     | 1.0.3        |
| claide                                     | 1.0.3        |
| colored                                    | 1.2          |
| colored2                                   | 3.1.2        |
| highline                                   | 1.7.10       |
| commander-fastlane                         | 4.4.6        |
| declarative                                | 0.0.10       |
| declarative-option                         | 0.1.0        |
| digest-crc                                 | 0.5.1        |
| unf_ext                                    | 0.0.7.7      |
| unf                                        | 0.1.4        |
| domain_name                                | 0.5.20190701 |
| dotenv                                     | 2.7.5        |
| emoji_regex                                | 1.0.1        |
| excon                                      | 0.73.0       |
| multipart-post                             | 2.0.0        |
| faraday                                    | 1.0.1        |
| http-cookie                                | 1.0.3        |
| faraday-cookie_jar                         | 0.0.6        |
| faraday_middleware                         | 1.0.0        |
| fastimage                                  | 2.1.7        |
| gh_inspector                               | 1.1.3        |
| jwt                                        | 2.1.0        |
| memoist                                    | 0.16.2       |
| multi_json                                 | 1.14.1       |
| os                                         | 1.1.0        |
| signet                                     | 0.14.0       |
| googleauth                                 | 0.12.0       |
| httpclient                                 | 2.8.3        |
| mini_mime                                  | 1.0.2        |
| uber                                       | 0.1.0        |
| representable                              | 3.0.4        |
| retriable                                  | 3.1.2        |
| google-api-client                          | 0.38.0       |
| google-cloud-env                           | 1.3.2        |
| google-cloud-errors                        | 1.0.1        |
| google-cloud-core                          | 1.5.0        |
| google-cloud-storage                       | 1.26.2       |
| json                                       | 2.3.0        |
| mini_magick                                | 4.10.1       |
| multi_xml                                  | 0.6.0        |
| plist                                      | 3.5.0        |
| rubyzip                                    | 1.3.0        |
| security                                   | 0.1.3        |
| naturally                                  | 2.2.0        |
| simctl                                     | 1.6.8        |
| slack-notifier                             | 2.3.2        |
| terminal-notifier                          | 2.0.0        |
| unicode-display_width                      | 1.7.0        |
| terminal-table                             | 1.8.0        |
| tty-screen                                 | 0.8.0        |
| tty-cursor                                 | 0.7.1        |
| tty-spinner                                | 0.9.3        |
| word_wrap                                  | 1.0.0        |
| nanaimo                                    | 0.2.6        |
| xcodeproj                                  | 1.16.0       |
| rouge                                      | 2.0.7        |
| xcpretty                                   | 0.3.0        |
| xcpretty-travis-formatter                  | 1.0.0        |
| fastlane-plugin-firebase_app_distribution  | 0.1.4        |
| fastlane-plugin-run_tests_firebase_testlab | 0.3.1        |
</details>

*generated on:* **2020-06-02**
</details>

**Comment by TheDancerCodes** [comment]
I have updated Fastlane from **2.149.1** to **2.150.3** (the latest version) but I am still facing the same issue.

`[!] Version '1.0.1' for 'com.thedancercodes.fastlanedroid.todoapp' does not seem to have any release notes. Nothing to download.`

**Comment by BurakDizlek** [comment]
I have same problem. is there any solution?

**Comment by TheDancerCodes** [comment]
@BurakDizlek I am still waiting on this as well.
None so far.
